pycharm-community.spec

```diff
@@ -141,72 +172,46 @@ desktop-file-install                          \
 %{buildroot}%{_datadir}/pycharm.desktop
 
 %files
-%defattr(-,root,root)
-%doc *.txt
-%doc license/
-%doc help/*.pdf
-%dir %{_datadir}/%{name}
 %{_datadir}/applications/pycharm.desktop
 %{_datadir}/mime/packages/%{name}.xml
 %{_datadir}/pixmaps/pycharm.png
 %{_datadir}/appdata/pycharm-community.appdata.xml
-%{_javadir}/%{name}/*
-%dir %{_javadir}/%{name}/%{plugins_dir}/git4idea/*
-%{_javadir}/%{name}/%{plugins_dir}/git4idea/*
-%dir %{_javadir}/%{name}/%{plugins_dir}/github/*
-%{_javadir}/%{name}/%{plugins_dir}/github/*
-%dir %{_javadir}/%{name}/%{plugins_dir}/hg4idea/*
-%{_javadir}/%{name}/%{plugins_dir}/hg4idea/*
-%dir %{_javadir}/%{name}/%{plugins_dir}/cvsIntegration/*
-%{_javadir}/%{name}/%{plugins_dir}/cvsIntegration/*
-%dir %{_javadir}/%{name}/%{plugins_dir}/ipnb/*
-%{_javadir}/%{name}/%{plugins_dir}/ipnb/*
-%dir %{_javadir}/%{name}/%{plugins_dir}/python-rest/*
-%{_javadir}/%{name}/%{plugins_dir}/python-rest/*
-%dir %{_javadir}/%{name}/%{plugins_dir}/rest/*
-%{_javadir}/%{name}/%{plugins_dir}/rest/*
-%dir %{_javadir}/%{name}/%{plugins_dir}/settings-repository/*
-%{_javadir}/%{name}/%{plugins_dir}/settings-repository/*
-%dir %{_javadir}/%{name}/%{plugins_dir}/svn4idea/*
-%{_javadir}/%{name}/%{plugins_dir}/svn4idea/*
-%dir %{_javadir}/%{name}/%{plugins_dir}/tasks/*
-%{_javadir}/%{name}/%{plugins_dir}/tasks/*
-%dir %{_javadir}/%{name}/%{plugins_dir}/terminal/*
-%{_javadir}/%{name}/%{plugins_dir}/terminal/*
-%exclude %{_javadir}/%{name}/%{plugins_dir}/{BashSupport,CppTools,idea-markdown}/*
-%exclude %{_javadir}/%{name}/%{plugins_dir}/{intellij-ansible,markdown,gitlab-integration-plugin}/*
-%exclude %{_javadir}/%{name}/%{plugins_dir}/{Go,IdeaVim,idea-multimarkdown,editorconfig,ini4idea}/*
+%{_javadir}/%{name}
```

Comment:
I guess, %dir %{_javadir}/%{name} should be there. Package has to own the directory.